packages/core/src/server/auth/token.ts

```diff
@@ -73,6 +73,28 @@ export class OAuth {
       }),
     });
 
+    if (response.status === 404) {
+      // Check for the deprecated authorize endpoint.
+      response = await fetch(`${wpUrl}/wp-json/wpac/v1/authorize`, {
+        headers: {
+          'Content-Type': 'application/json',
+          'x-wpe-headless-secret': apiClientSecret,
+        },
+        method: 'POST',
+        body: JSON.stringify({
+          code,
+          refreshToken: this.getRefreshToken(),
+        }),
+      });
+
+      if (response.status !== 404 && process.env.NODE_ENV !== 'production') {
```

Comment:
Is this worthwhile? If so, is there a better environment variable we can check? I was thinking something that more explicitly asks for logging or debug info would be better, but don't know if such a thing exists.

Reply:
we do have the [`disableLoggin`](https://github.com/wpengine/faustjs/blob/canary/packages/core/src/config/config.ts#L90) config option. This could be appropriate here.